_components/Feedback.tsx

```diff
@@ -138,6 +157,20 @@ export default function Feedback({ file }: { file: string | undefined }) {
                   </button>
                 </div>
               </div>
+              {modifiedDate &&
+                (
+                  <div class="mt-4">
+                    This page was last modified on{" "}
+                    <time datetime={modifiedDate.toString()}>
```

Comment:
```suggestion
                    <time datetime={modifiedDate.toISOString()}>
```

> The toString() method of [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) instances returns a string representing this date interpreted in the local timezone.

[`toString()` returns localized date;](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toString) but `<time>` requires [datetime in ISO8601 format](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/time#valid_datetime_values).